Geek-Collectors-Network/geek-collectors-network
# Implement user locations

### Summary
 - Add country, region, and city columns to User DB model
 - Generate dummy location fields and add to dummy data
 - Update /user/profile endpoint to return location fields
 - Update /user/profile patch endpoint to accept location fields
 - Update edit profile page to include country, region, and city fields

### Testing
1. Build the latest frontend: `npm run -w apps/frontend build`
2. Spin up the db: `make res`
3. Start the api server: `npm run -w apps/api start:dev`
4. Navigate to: http://127.0.0.1:3000/
6. Login with any dummy account
7. Verify you can see populated country, region, and city fields: http://127.0.0.1:3000/profile/edit
